md_api/src/main/java/com/wipro/ats/bdre/md/api/TermJob.java

```diff
@@ -78,10 +83,8 @@ public TermJobInfo execute(String[] params) {
             //The TermJob completes even if sending message fails
             try {
                 BasicConfigurator.configure();
-                StatusNotification statusNotification =new StatusNotification(termMessage, MDConfig.getProperty("status-notification.term-queue"));
-                LOGGER.info(statusNotification.toString());
+                new StatusNotification(termMessage, MDConfig.getProperty("status-notification.term-queue"));
```

Comment:
![CRITICAL](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-critical.png) Either remove this useless object instantiation of class "StatusNotification" or use it [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1848)